compiler/base.ts

```diff
@@ -959,7 +959,7 @@ export class LineCompiler implements InstrVisitor<void>, ExprVisitor<void> {
     this.emitConstant(str.value);
   }
 
-  visitPathLiteralExpr(path: PathLiteral): void {
+  visitShellLiteralExpr(path: ShellLiteral): void {
```

Comment:
This would probably store the value as a string. Though, I could potentially introduce a new value type...

Reply:
Doing this for now.